src/TimeTracker.Library/Services/TimeEntryService.cs

```diff
@@ -103,17 +100,49 @@ public async Task<double> DeleteHours(DateTime date)
                 return 0;
             }
             
-            var hoursDeleted = timeEntries.Sum(x=>x.Hours);
+            return await DeleteHoursFromDb(timeEntries);
+        }
+
+        public async Task<double> DeleteHoursForTimeEntryType(DateTime date, TimeEntryTypeEnum timeEntryType)
```

Comment:
consider renaming `date` to `entryDate` to be more specific

Reply:
nevermind, just saw we're using `date` everywhere and don't think it'll add much to rename